ddp-workspace/projects/ddp-angio/src/app/app-routing.module.ts

```diff
@@ -2,249 +2,23 @@ import { NgModule } from '@angular/core';
-            AuthGuard
-        ],
-        data: {
-            activityGuid: 'ANGIOLOVEDONE',
-            createActivityInstance: true
-        }
-    },
-    {
-        path: 'consent',
-        component: ActivityPageComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard,
-            AuthGuard
-        ],
-        data: {
-            activityGuid: 'ANGIOCONSENT'
-        }
-    },
-    {
-        path: 'release-survey',
-        component: ActivityPageComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard,
-            AuthGuard
-        ],
-        data: {
-            activityGuid: 'ANGIORELEASE'
-        }
-    },
-    {
-        path: 'dashboard',
-        component: DashboardComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard,
-            AuthGuard
-        ]
-    },
-    {
-        path: 'auth',
-        component: Auth0CodeCallbackComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'activity/:id',
-        component: ActivityComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard,
-            AuthGuard
-        ]
-    },
-    {
-        path: 'activity-link/:id',
-        component: ActivityLinkComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard,
-            AuthGuard
-        ]
-    },
-    {
-        path: 'login-landing',
-        component: LoginLandingComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'login-landing/:mode',
-        component: RedirectToAuth0LoginComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'count-me-in',
-        component: WorkflowStartActivityComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard
-        ]
-    },
-    {
-        path: 'about-us',
-        component: AboutUsComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'error',
-        component: ErrorComponent
-    },
     {
         path: 'data-release',
         component: DataReleaseComponent,
         canActivate: [IrbGuard]
     },
-    {
-        path: 'more-details',
-        component: MoreDetailsComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'stay-informed',
-        component: StayInformedComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'loved-one-thank-you',
-        component: LovedOneThankYouComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'international-patients',
-        component: InternationalPatientsComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'password-reset-done',
-        component: RedirectToLoginLandingComponent,
-        canActivate: [IrbGuard]
-    },
-    {
-        path: 'session-expired',
-        component: SessionExpiredComponent,
-        canActivate: [
-            IrbGuard,
-            BrowserGuard
-        ]
-    },
-    {
```

Comment:
When running locally, I had to add the `PasswordComponent` stuff back in.  It's an edge case since the IRB password will be disabled in the real world, but is there any harm in keeping the password route open?

Reply:
fixed